docs/src/misc.rst

```diff
@@ -271,6 +271,12 @@ API
     and :man:`inet_pton(3)`. On success they return 0. In case of error
     the target `dst` pointer is unmodified.
 
+.. c:function:: char* uv_if_indextoname(unsigned int ifindex, char* ifname)
+
+    Cross-platform IPv6-capable implementation of :man:`if_indextoname(3)`.
+    Writes the result to `ifname`, which must be at least `IFNAMSIZ` long.
```

Comment:
On linux the constant is `IF_NAMESIZE` so we could add a `UV_IF_NAMESIZE` constant to avoid platform inconsistencies